integration/skydb.test.ts

```diff
@@ -137,9 +136,9 @@ describe(`SkyDB end to end integration tests for portal '${portal}'`, () => {
     // TODO: Should getFileContent return `null` on 404?
     try {
       await client.getFileContent(entryLink);
-      throw new Error("getFileContent should not have succeeded");
+      throw new Error("'getFileContent' should not have succeeded");
     } catch (err) {
-      expect((err as AxiosError).response?.status).toEqual(404);
+      expect((err as ExecuteRequestError).responseStatus).toEqual(404);
```

Comment:
maybe we should assert the type here and assert `instanceof` behaves as expected